vos/vos/vos.py

```diff
@@ -238,6 +247,7 @@ class Node(object):
 
     IVOAURL = "ivo://ivoa.net/vospace/core"
     VOSNS = "http://www.ivoa.net/xml/VOSpace/v2.0"
+    VOSVERSION = "2.1"
```

Comment:
Strings are inconsistent throughout the file.  I know both single and double quotes are supported, but we should pick one and stick with it throughout, unless there is a single quote or double quote _in_ the string, then use the opposite (i.e. `"'red'"` instead of `'\'red\''`).

Reply:
I know. I've fixed some but they are way too many. One day I'll do a global replace.